Security-Operations/L4-Security-Analyst-Technical-Lead.md

```diff
@@ -0,0 +1,87 @@
+# L4: Security Analyst Technical Lead
+
+> _I possess expert-level knowledge and experience in cybersecurity, with the ability to lead advanced threat hunting and forensic analysis, mentor other analysts, and influence security strategy across the Security Analyst team. I am responsible for driving large-scale security initiatives, technical direction, and innovation to enhance the organisation's security posture._
+
+- **Planning Horizon**: 6-12 months.
+- **Impact Radius**: Team (5-15).
+- **Evaluation**: Manager or Director.
+- **Responsibility and Direction Needed**: Takes brand new ideas from inception to production with limited supervision.
+
+## 🦉 Domain expertise
+
+- I am a subject matter expert in my domain and recognised as such within the organisation.
+- I am confident diving head-first into risky or contentious situations with relatively little upfront information and oversight.
+- I proactively identify and work with cross-functional teams to mitigate risks and product vulnerabilities.
+- I stay up to date on the latest trends, platform features, and critical vendor technologies to anticipate and mitigate potential threats.
+- I provide expert-level guidance and support during security incident investigations and post-mortems.
+- I ensure that our Security Information and Event Management (SIEM) system is optimised to ensure we only receive actionable alerts.
```

Comment:
Idea:
- Actionable alerts - yes!
- Identify gaps/breakages and rectify quickly